packages/web/tailwind.config.js

```diff
@@ -223,6 +226,7 @@ module.exports = {
           5: "4.75rem",
           6: "5rem",
         },
+        "300px": "300px",
```

Comment:
With tailwind, you can almost always. use arbitrary values as needed, inlined: https://tailwindcss.com/docs/padding#arbitrary-values.

Stuff should only be added to the tailwind config if it's reused or expected to be reused.